README.md

```diff
@@ -1 +1,56 @@
-# WeatherApp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Weather App
+
+> A weather app: is an app that pulls data from the OpenWeatherMap API to allow users to search for and view the forecast in cities worldwide. Built with JavaScript using webpack and async functions.
+
+![](./assets/images/weather.png)
+
+
+> A Weather App is part of Full-Stack JavaScript Curriculum in the [the Odin Project’s](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript/lessons/weather-app).
+
+## Built With
+
+- JavaScript (Async functions)
+- Webpack
+- API (OpenWeathermap)
+- HTML5
+- CSS3
+- Bootstrap
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Emmyn5600/WeatherApp/weather/feat/dist/index.html)
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+    Step 1: git clone https://github.com/Emmyn5600/WeatherApp.git
+    
+    Step 2: open live-server
```

Comment:
- add instruction for installing packages with npm
- add the command for changing directory after cloning and git branch